test/unit/store/toast-store.js

```diff
@@ -127,4 +127,13 @@ describe('ToastStore', () => {
         assert.match(ToastStore.getToasts()[3].text, /Created a new Car Radio/);
     });
 
+    it('should add a toast message when creating a new item', () => {
+        Dispatcher.handleAction('ITEM_CREATED', {
+            address: 'iGwEZUvfA',
+            modelName: 'Resistor'
+        });
+        assert.lengthOf(ToastStore.getToasts(), 4);
+        assert.match(ToastStore.getToasts()[3].text, /Created a new Resistor/);
```

Comment:
Why is this not asserting the text vs "Created a new Resistor item!"? Doesn't the toast for making a new item add the extra "item!"?